tests/unit/test_leo.py

```diff
@@ -15,6 +16,7 @@
 from tests import BIGQUERY_ACCESS, OFFLINE
 
 IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
+PYCONTRAILS_SKIP_LEO_TESTS = bool(os.getenv("PYCONTRAILS_SKIP_LEO_TESTS"))
```

Comment:
@thabbott I added this because the unit tests were taking too long for me to run locally. I'm happy to revert if you think this is a bad idea. (I can always set a `GITHUB_ACTIONS` environment variable as a hack.)

Reply:
This makes me wonder at what point we start using a conf file....

Reply:
I think it makes sense to have something besides GITHUB_ACTIONS that you can use to skip tests locally. Maybe Marc has ideas for a different way to implement conditional skipping? We could bring this up at standup to make sure there's not a better approach.